MapboxNavigation/DefaultStyle.swift

```diff
@@ -107,5 +107,14 @@ open class DefaultStyle: Style {
         
         routeCasingColor = .defaultRouteCasing
     }
+}
+
+/**
+ `NightStyle` is default night style for Mapbox Navigation SDK. Only will be applied if `automaticallyAdjustStyleForSunPosition` and `isDaytime` are true.
```

Comment:
`isDaytime` is an internal property, so it shouldn’t be mentioned in public documentation.